docs/concepts/artifact.mdx

```diff
@@ -0,0 +1,239 @@
+    "digest": "sha256:34bf03806938a59ee7dc3e2c33e314d0eaef573313ff9dcc677113502d568523",
+    "size": 145
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.cncf.helm.chart.content.v1.tar+gzip",
+      "digest": "sha256:4d80464e9d8e9f3ba92e6ead6d3b5afd0532cb0a81e980599a0bced99fdc6e01",
+      "size": 3763
+    }
+  ]
+}
+```
+
+### Annotations only artifacts
+
+Artifacts may store metadata in the manifest as annotations and do not have a config or blobs.
+For these the `artifactType` property is used to declare the type of the artifact.
+The config property is required in the manifest for maximum compatiblity an empty layer is also created.
+as per the  [empty descriptors guidance][empty-descriptor].  
+
+```json
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "artifactType": "application/vnd.example+type",
+  "config": {
+    "mediaType": "application/vnd.oci.empty.v1+json",
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+    "size": 2
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.oci.empty.v1+json",
+      "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+      "size": 2
+    }
+  ],
+  "annotations": {
+    "oci.opencontainers.image.created": "2023-01-02T03:04:05Z",
+    "com.example.info": "useful-info"
+  }
+}
+```
+
+### Deciding on artifactType or config.mediaType
+
+Considering the above types of artifacts, the decision tree below should help
+determine what fields to set when creating an artifact.
+
+```mermaid
+graph TD
+    A{Artifact has at least<br>one file or blob?}
+    B{Artifact has additional<br>JSON metadata?}
+    C{Artifact requires <br>different artifactType<br>from config.mediaType?}
+    D[Specify artifactType,<br>set config and layers<br>to empty descriptors.]
+    E[Specify artifactType, include<br>artifact in layers, set<br>config to empty descriptor.]
+    F[Specify config with metadata,<br>include artifact in layers,<br>do not set artifactType.]
+    G[Specify artifactType to common<br>value, specify config with<br>metadata, include artifact in layers.]
+
+    A -- No --> D
+    A -- Yes --> B
+    B -- No --> E
+    B -- Yes --> C
+    C -- No --> F
+    C -- Yes --> G
+
+```
+
+### Harnessing Image Indexes
+
+One of the key advantages of OCI artifacts is their ability to utilize [Image Indexes][image-index],
+allowing the bundling of various OCI artifacts.
+
+The Image Index is a higher-level construct in the OCI Image Specification that can point to
+multiple Image Manifests, each suitable for a different platform or architecture or filtered by annotation.
+It helps to group related artifacts together, providing a single entry-point for accessing any specific
+artifact depending on the required condition.
+
+## Best Practices and Limitations
+
+While working with OCI artifacts, keep in mind that not all client tools or registries
```

Comment:
Suggest this gets split up a bit as there's a difference for creation and consumption. 
While working with OCI artifacts, keep in mind that not all client tools or registries have added 1.1 support for the Image and Distribution specifications. 

- When consuming OCI Artifacts, always check the existence of the `artifactType` and use it when available. Fallback to the `config.mediaType` when it doesn't exist.
- When creating new OCI Artifacts, set the `artifactType` property to define the usage. Only create a unique `config.mediaType` if you need a config object. Otherwise, use the `"config.mediaType": "application/vnd.oci.empty.v1+json"`
-

Reply:
I would like to get the first draft of this merged and update best practices post.